index.html

```diff
@@ -181,13 +181,17 @@ <h2>Contact me</h2>
                 If you have an application you are interested in developing, 
                 a feature that you need built or a project that needs coding. 
                 i'd love to help with it.</p>
-            <form action="https://formspree.io/f/xvolnqro" methop="post" class="text-me">
-                <input type="text" class="simple" placeholder="User name" maxlength="30" required>
-                <input type="email" class="simple" placeholder="Email" required>
-                <textarea class="text-area" placeholder="comment" required></textarea>
-                <button type="submit" class="project">Get in touch</button>
-            </form>       
+            <form class="text-me" id="#form" method="post" action="https://formspree.io/f/xzboqvlk">
```

Comment:
- [ ] [OPTIONAL] You should use the JavaScript & HTML/CSS linters instead of just HTML/CSS linters, so the linters can check the JavaScript code also